internal/rukpak/convert/registryv1_test.go

```diff
@@ -167,22 +169,38 @@ var _ = Describe("RegistryV1 Suite", func() {
 		Context("Should generate objects successfully based on target namespaces", func() {
 			var (
 				svc             corev1.Service
-				csv             v1alpha1.ClusterServiceVersion
+				baseCSV         v1alpha1.ClusterServiceVersion
 				watchNamespaces []string
 			)
 
 			BeforeEach(func() {
-				csv = v1alpha1.ClusterServiceVersion{
+				baseCSV = v1alpha1.ClusterServiceVersion{
```

Comment:
would it be worth adding a comment here to say that this will be deep copied and modified by each test case?
```suggestion
                                // base CSV definition that each test case will deep copy and modify 
				baseCSV = v1alpha1.ClusterServiceVersion{
```